test/functional/http/get/accounts.js

```diff
@@ -2,319 +2,268 @@
-							node.expect(res).to.have.property('status').to.equal(200);
-							node.expect(res).to.have.nested.property('body.accounts').that.is.an('array').to.have.length(5);
-							var accountBalances = _.map(res.body.accounts, function (account) {
-								return Number(account.balance);
-							});
-							node.expect(_.map(_.map(res.body.accounts, 'balance'), Number)).eql(_.sortBy(accountBalances).reverse());
-						});
-					});
-
-					it('using sort = balance:asc should sort accounts by balance in ascending order', function () {
-						return getAccountsPromise('limit=' + 5 + '&sort=balance:asc').then(function (res) {
-							node.expect(res).to.have.property('status').to.equal(200);
-							node.expect(res).to.have.nested.property('body.accounts').that.is.an('array').to.have.length(5);
-							var accountBalances = _.map(res.body.accounts, function (account) {
-								return Number(account.balance);
-							});
-							node.expect(_.map(_.map(res.body.accounts, 'balance'), Number)).eql(_.sortBy(accountBalances));
-						});
-					});
-				});
+		describe('offset', function () {
 
-				describe('username', function () {
-
-					it('using sort = username should sort accounts by username in default ascending order', function () {
-						return getAccountsPromise('limit=' + 5 + '&sort=username').then(function (res) {
-							node.expect(res).to.have.property('status').to.equal(200);
-							node.expect(res).to.have.nested.property('body.accounts').that.is.an('array').to.have.length(5);
-							node.expect(_.map(res.body.accounts, 'delegate.username')).eql(_.map(_.sortBy(res.body.accounts, 'delegate.username'), 'delegate.username'));
-						});
-					});
-
-					it('using sort = username:desc should sort accounts by balance in descending order', function () {
-						return getAccountsPromise('limit=' + 5 + '&sort=username:desc').then(function (res) {
-							node.expect(res).to.have.property('status').to.equal(200);
-							node.expect(res).to.have.nested.property('body.accounts').that.is.an('array').to.have.length(5);
-							node.expect(_.map(res.body.accounts, 'delegate.username')).eql(_.map(_.sortBy(res.body.accounts, 'delegate.username').reverse(), 'delegate.username'));
-						});
-					});
-
-					it('using sort = username:asc should sort accounts by balance in ascending order', function () {
-						return getAccountsPromise('limit=' + 5 + '&sort=username:asc').then(function (res) {
-							node.expect(res).to.have.property('status').to.equal(200);
-							node.expect(res).to.have.nested.property('body.accounts').that.is.an('array').to.have.length(5);
-							node.expect(_.map(res.body.accounts, 'delegate.username')).eql(_.map(_.sortBy(res.body.accounts, 'username'), 'delegate.username'));
-						});
-					});
+			it('using offset = -1 should return error', function () {
+				return accountsEndpoint.makeRequest({offset:  -1}, 400).then(function (res) {
```

Comment:
double spaces in some of these objects. Is there a reason for it?

Reply:
Done.